docs/src/views/Architecture.tsx

```diff
@@ -0,0 +1,224 @@
+import * as React from 'react'
+import { NavLink } from 'react-router-dom'
+
+import { Button, Header, Divider } from '@stardust-ui/react'
+
+import DocPage from '../components/DocPage'
+import CodeSnippet from '../components/CodeSnippet'
+
+export default () => (
+  <DocPage title="Architecture">
+    <Header as="h2">Overview</Header>
+    <p>
+      There are two main drivers in the architecture decisions: <strong>accessibility</strong> and{' '}
+      <strong>theming</strong>.
+    </p>
+    <p>Stardust UI project consists of two parts:</p>
+    <ul>
+      <li>
+        <strong>Toolkit (Framework)</strong>, which allows to create any component: primitive or
+        composed.
+      </li>
+      <li>
+        <strong>Component Library</strong>, which is the set of primitive and composed components.
+      </li>
+    </ul>
+
+    <Header as="h2">Toolkit (Framework)</Header>
+    <p>
+      The responsibility of the toolkit part of the project is to provide an easy way to create UI
+      components and encapsulate theming and accessibility logic. This part of the project is almost
+      UI framework agnostic and written in pure TypeScript. Stardust UI uses this toolkit to create
+      our own set of UI Components and allows other consumers to create their own primitive or
+      composed components. This architecture allows to combine Stradust UI primitive or composed
+      components with the custom primitive or composed components made by third parties, who were
+      using the toolkit to create their own components.
+    </p>
+    <p>The toolkit combines the following parts:</p>
+    <ul>
+      <li>
+        <strong>Component Logic</strong> - the framework core logic, which couples all other parts
```

Comment:
We have no tools for component logic.  Component logic is implemented by each component on an as needed basis. I don't think we should include this in the docs.